src/waypoint.jsx

```diff
@@ -370,11 +372,5 @@ Waypoint.above = ABOVE;
 Waypoint.below = BELOW;
 Waypoint.inside = INSIDE;
 Waypoint.invisible = INVISIBLE;
-Waypoint.getWindow = () => {
```

Comment:
It looks like this was added by 4e22e27e9290bba61f4c133509cc8e2611e72079 but I'm not really sure why. @trotzig do you happen to know?

Reply:
I believe it was only added to allow this mock: 

https://github.com/brigade/react-waypoint/commit/4e22e27e9290bba61f4c133509cc8e2611e72079#diff-27fadc9a7e55db698d635052018a532aR424

Reply:
Cool, yeah I think this is okay to remove!